lib/grpc-api-assistant/grpc_steps.rb

```diff
@@ -222,3 +242,7 @@
   value = GrpcHelpers::fetch_from_grpc_with_shorthand(field_path, @grpc_response)
   expect(value).not_to be
 end
+
+Given(/^I wait (\d+) second(?:s)?$/) do |seconds|
```

Comment:
This is probably the simplest way to test whether appropriate conditions can bypass the cache - open to other ideas? Maybe a month forward or backward?

Reply:
This seems like a reasonable step, though, given the regex, milliseconds might be better.  I think most of our use cases can get away with a very, very short wait (e.g. 100ms even) and this regex, I think, wouldn't allow for decimals, only whole numbers.

At some point we should probably update these steps to use the latest Cucumber type captures, but not today.